libtmux/common.py

```diff
@@ -552,7 +552,7 @@ def get_version() -> LooseVersion:
             )
         raise exc.VersionTooLow(proc.stderr)
 
-    version = proc.stdout[0].split("tmux ")[1]
+    version = re.search('[0-9]{1}.[0-9]', proc.stdout[0]).group(0)
```

Comment:
`poetry run mypy .`

`libtmux/common.py:555: error: Item "None" of "Optional[Match[str]]" has no attribute "group"`

This will need null-checks to handle empty matches.  I haven't tested, but it'd be something like

```suggestion
    match = re.search('[0-9]{1}.[0-9]', proc.stdout[0])
    assert match is not None
    version = match.group(0)
```